analysis/src/analyzer.js

```diff
@@ -28,9 +28,14 @@ class AnalyzerRunner {
 
       if (inputs == null || inputs.length === 0) {
         resolve({});
+        // TODO: fall back to package analysis
       } else {
         Promise.all(inputs.map((i) => analyzer.analyze(i))).then(function(documents) {
-          resolve(generateAnalysis(documents, ''));
+          var allDocuments = [];
+          documents.forEach(function(doc) {
+            allDocuments = allDocuments.concat(Array.from(doc.getByKind('document', {imported: true, externalPackages: false})));
```

Comment:
Yes.

Based on offline convos, we were going to ignore main documents & search the entire package, but this turned out to include the wrong things for packages like `app-layout`, which include all sorts of elements for testing, demos, sites etc.

This instead analyses all the main documents, and only includes everything they import while still in the same package.